src/Client.ts

```diff
@@ -187,7 +200,11 @@ export default class Client {
     const apiClient = createApiClientFromOptions(options)
     const keys = await loadOrCreateKeysFromOptions(options, wallet, apiClient)
     apiClient.setAuthenticator(new Authenticator(keys.identityKey))
-    const client = new Client(keys, apiClient)
+    const backupClient = await Client.setupBackupClient(
+      keys.identityKey,
```

Comment:
How is this going to work in a world where the `Client` doesn't have access to the IdentityKey itself? Once the KeyStore work goes in, the `IdentityKey` will only live inside the KeyStore, and the KeyStore may be remote (for example, MM Snap)